README.md

```diff
@@ -9,6 +9,23 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
   git clone https://github.com/thewca/worldcubeassociation.org
   cd worldcubeassociation.org
   ```
+
+We can run the application in two ways:
+- [Run using Docker](#run-using-docker)
```

Comment:
Add a note that Docker is strongly recommended, and that choosing a manual installation happens "at your own risk" and we may not be able to assist with detailed troubleshooting

Reply:
Okay done. Added the 'risk' sentence to rails steps.

Reply:
In docker, should we also add a note that while committing, they have to use `--no-verify` to skip the precommit? Because the precommit requires rails to be installed.

Reply:
No, our precommit hook has been unmaintained for ages. I'd rather recommend against installing it into local environments (we use it mainly for CI...)